microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -59,8 +59,10 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-13-networking"]
 //=== Networking
 
-//[id="microshift-4-13-storage"]
-//=== Storage
+[id="microshift-4-13-storage"]
+=== Storage
+
+* {product-title} configures LVMS more dynamically based on the Volume Groups present. For more details, read the xref:../microshift_storage/microshift-storage-plugin-overview.adoc#lvms-volume-group-name_microshift-storage-plugin-overview[LVMS system requirements] section of this documentation.
```

Comment:
Can you remove the _more_ here_? I tripped over _more dynamically based_, reading it as _more dynamically-based_ (if that makes sense).